okhttp/src/main/java/io/grpc/okhttp/OkHttpChannelBuilder.java

```diff
@@ -270,6 +274,29 @@ SSLSocketFactory createSocketFactory() {
     }
   }
 
+  private static SSLContext getSslContext() throws NoSuchAlgorithmException {
+    if (GrpcUtil.IS_RESTRICTED_APPENGINE) {
+      try {
+        SSLContext tlsContext = SSLContext.getInstance("TLS", Platform.get().getProvider());
+        KeyStore keyStore = KeyStore.getInstance(KeyStore.getDefaultType());
+        keyStore.load(null, null);
+        KeyManagerFactory keyManagerFactory =
+            KeyManagerFactory.getInstance(KeyManagerFactory.getDefaultAlgorithm());
+        keyManagerFactory.init(keyStore, null);
+        TrustManagerFactory trustManagerFactory =
+            TrustManagerFactory.getInstance(TrustManagerFactory.getDefaultAlgorithm());
+        trustManagerFactory.init((KeyStore) null);
+        tlsContext.init(
+            null,
```

Comment:
Every literal in this function needs to be documented.  (what is null?  what is "SHA1PRNG")